src/v0/destinations/marketo/util.js

```diff
@@ -34,29 +36,28 @@ const MARKETO_ABORTABLE_CODES = [
 ];
 const MARKETO_THROTTLED_CODES = ['502', '606', '607', '608', '615'];
 
-const RECORD_LEVEL_ABORTBALE_ERRORS = [
-  '1001',
-  '1002',
-  '1003',
-  '1004',
-  '1005',
-  '1006',
-  '1007',
-  '1008',
-  '1011',
-  '1013',
-  '1014',
-  '1015',
-  '1016',
-  '1017',
-  '1018',
-  '1021',
-  '1026',
-  '1027',
-  '1028',
-  '1036',
-  '1049',
-];
+// Keeping here for reference const RECORD_LEVEL_ABORTBALE_ERRORS = [
+//   '1001',
+//   '1002',
+//   '1003',
+//   '1004',
+//   '1005',
+//   '1006',
+//   '1007',
+//   '1008',
+//   '1011',
+//   '1013',
+//   '1014',
+//   '1016',
+//   '1017',
+//   '1018',
+//   '1021',
+//   '1026',
+//   '1027',
+//   '1028',
+//   '1036',
+//   '1049',
+// ];
```

Comment:
Why are we commenting these?

Reply:
We are by default already converting these to 400 so these are not used anymore but kept them as it is for future ref as there remains some confusion about marketo status codes